src/pages/gig-detail-page.html

```diff
@@ -42,6 +43,7 @@
 
   <template is="dom-if" if="[[gig]]">
     <social-meta title="[[gig.title]] job at [[gig.posterCompany]] - Polygigs" content="[[gig.posterCompany]] in [[gig.location]] is hiring for [[gig.title]]"></social-meta>
+    <page-title base-title="Polygigs" title="[[gig.title]] job at [[gig.posterCompany]]" direction="reversed"></page-title>
```

Comment:
This doesn't appear to be working for search engines, which is strange because the meta trick worked.